synthwave-theme.el

```diff
@@ -1,139 +1,400 @@
+
+       (link `(:weight normal :underline t :foreground ,synth-light-blue))
+       (link-no-underline `(:weight normal :foreground ,synth-light-blue))
+       (link-visited `(:weight normal :underline t :foreground ,synth-deep-purple))
+       (link-visited-no-underline `(:weight normal :foreground ,synth-deep-purple))
+
+       (mail-header-name `(:family "Sans Serif" :weight normal :foreground ,synth-light-gold))
+       (mail-header-other `(:family "Sans Serif" :slant normal :foreground ,synth-gold))
+       (mail-header-subject `(:family "Sans Serif" :weight bold :foreground ,synth-green))
+       (mail-read `(:foreground ,synth-dark-gold))
+       (mail-read-high `(:foreground ,synth-deep-gold))
+       (mail-ticked `(:foreground ,synth-deep-red))
+       (mail-to `(:family "Sans Serif" :underline nil :foreground ,synth-pale-gold))
+       (mail-unread `(:weight bold :foreground ,near-near-white))
+       (mail-unread-high `(:weight bold :foreground ,synth-gold))
+
+       (match `(:weight bold :background ,synth-gold)) ; occur patterns + match in helm for files + match in Org files.
+
+       (ol1 `(,@(synthwave-scale-font synthwave-scale-outline-headlines 1.4) :weight bold :overline ,synth-deep-red :foreground ,synth-pink :background ,near-near-black))
+       (ol2 `(,@(synthwave-scale-font synthwave-scale-outline-headlines 1.2) :weight bold :overline ,synth-deep-purple :foreground ,synth-pale-purple :background ,near-near-black))
+       (ol3 `(:height 1.0 :weight bold :foreground ,synth-dark-pink :background ,near-near-black))
+       (ol4 `(:height 1.0 :weight bold :slant normal :foreground ,synth-deep-gold))
+       (ol5 `(:height 1.0 :weight bold :slant normal :foreground ,synth-dark-gold))
+       (ol6 `(:height 1.0 :weight bold :slant italic :foreground ,synth-deep-orange))
+       (ol7 `(:height 1.0 :weight bold :slant italic :foreground ,synth-deep-red))
+       (ol8 `(:height 1.0 :weight bold :slant italic :foreground ,synth-dark-red))
+
+       (paren-matched `(:background ,synth-dark-green))
+       (paren-unmatched `(:weight bold :underline ,synth-dark-red :foreground ,synth-deep-red :background ,near-near-black))
+
+       (tab `(:foreground ,near-near-black :background ,synth-dark-blue))
+       (trailing `(:foreground "#E8E8E8" :background ,synth-dark-blue))
+      )
+
+  (custom-theme-set-faces
+   'synthwave
+   ;; default faces
+   `(cursor ((,class (:background ,cursor-color :foreground "black"))))
+   `(default ((,class (:background ,default-background :foreground ,near-white))))
+
+   `(bold ((,class (:weight bold :foreground "white"))))
+   `(bold-italic ((,class (:weight1 bold :slant italic :foreground "white"))))
+   `(italic ((,class (:slant italic :foreground "white"))))
+   `(underline ((,class (:underline t :background ,near-near-black :foreground ,near-white))))
+
+   ;; highlight faces
+   `(fringe ((,class (:underline t :background ,synth-orange :foreground ,synth-blue))))
```

Comment:
I dislike the fringe color combination, I find the orange distracting. I would prefer a more muted tone that complements the modeline colorscheme. I see the value in an alternative background color, but want it to be more subtle. Foreground color may be aggressive or moderate.